### ISSUE TITLE
why the generated masks are different from that in the original paper?

### ISSUE THREAD

**Issue by micklexqg** [naoto0804/pytorch-inpainting-with-partial-conv#20]
thanks for your work. i note that the generated masks using the code are different from the the original paper.
the generated masks in the paper are as follows:
![image](https://user-images.githubusercontent.com/13776012/45661703-3b733e00-bb31-11e8-9981-71f269261d4e.png)

the generated masks using the code are as follows:
![image](https://user-images.githubusercontent.com/13776012/45661728-59d93980-bb31-11e8-9436-be52551a8c25.png)

do you follow the method of the paper or just use your own way for generating masks?

**Comment by naoto0804** [comment]
Thank you for visiting this repository!
I could not get the information enough to reproduce the masking, so I follow another [implementation](https://github.com/SeitaroShinagawa/chainer-partial_convolution_image_inpainting) for the mask generation. I would really appreciate it if you could tell me how to do that.

**Comment by micklexqg** [comment]
@naoto0804 , sorry for late reply. i am working on it. have you finished the training or are there any pre-trained models?

**Comment by naoto0804** [comment]
I have one model trained for 1000k (training -> fine-tuning)
By the way, as mentioned in #14, the authors finally make some part of their masks used in the paper publicly available. You might also try it.

**Comment by micklexqg** [comment]
thanks, can you share the pretrained model?

**Comment by naoto0804** [comment]
https://www.dropbox.com/s/34ygasw6f7dt77p/1000000.pth?dl=0

**Comment by micklexqg** [comment]
thanks!
